WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m

```diff
@@ -95,7 +95,6 @@ - (void)viewDidAppear:(BOOL)animated
 - (void)viewWillDisappear:(BOOL)animated
 {
     [super viewWillDisappear:animated];
-    [self stopLoading];
```

Comment:
Fyi, this was removed on purpose.  During my tests I found that it looks nicer if we let the preview keep loading even as this view disappears.

This can be tested by opening a preview, and switching to another section of the App (using the bottom tab bar).

- With the previous code, once you go back to "My Sites" the preview is still blank.
- With this change, the preview is properly loaded.

In both scenarios the preview is refreshed, but the latter looks much better IMHO.

Reply:
Good thinking! 👏 

And thank you for explaining.